torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1811,6 +1832,26 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
            decorators=[_wrap_maybe_warns("floor_divide is deprecated, and will be removed")],
            supports_autograd=False,
            ),
+    OpInfo('linalg.multi_dot',
```

Comment:
Not for this PR, but we should make a plan for the existing chain_matmul() tests like 

https://github.com/pytorch/pytorch/blob/f5617b0932c711ff4701904e9462a29fcb020d70/test/test_autograd.py#L2972